### ISSUE TITLE
test-suite build failure

### ISSUE THREAD

**Issue by bergmark** [stackbuilders/inflections-hs#10]
```
Preprocessing test suite 'test' for inflections-0.1.0.10...

test/Suite.hs:5:18:
    Could not find module ‘Text.InflectionsTest’
    Perhaps you meant
      Text.Inflections (needs flag -package-key inflections-0.1.0.10@infle_CQqKGdu9Gm59Y5luae0UI7)
      Text.Inflections (from inflections-0.1.0.10@infle_FVahW5jDpQl808ny9EqK7f)
      Text.Inflections.Data
    Use -v to see a list of the files searched for.

test/Suite.hs:6:18:
    Could not find module ‘Text.Inflections.Tests’
    Perhaps you meant
      Text.Inflections.Data
      Text.Inflections.Data
      Text.Inflections (needs flag -package-key inflections-0.1.0.10@infle_CQqKGdu9Gm59Y5luae0UI7)
    Use -v to see a list of the files searched for.

test/Suite.hs:7:18:
    Could not find module ‘Text.Inflections.UnderscoreTest’
    Perhaps you meant
      Text.Inflections.Underscore
      Text.Inflections.Underscore
    Use -v to see a list of the files searched for.

test/Suite.hs:8:18:
    Could not find module ‘Text.Inflections.OrdinalTest’
    Perhaps you meant
      Text.Inflections.Ordinal
      Text.Inflections.Ordinal
    Use -v to see a list of the files searched for.

test/Suite.hs:9:18:
    Could not find module ‘Text.Inflections.HumanizeTest’
    Perhaps you meant
      Text.Inflections.Humanize
      Text.Inflections.Humanize
    Use -v to see a list of the files searched for.

test/Suite.hs:10:18:
    Could not find module ‘Text.Inflections.TitleizeTest’
    Perhaps you meant
      Text.Inflections.Titleize
      Text.Inflections.Titleize
    Use -v to see a list of the files searched for.
Failed to install inflections-0.1.0.10
```

ping @jpvillaisaza

**Comment by jpvillaisaza** [comment]
@bergmark Thank you for opening this issue. I'll close it as soon as we upload version 0.2.0.0 to Hackage.

**Comment by jpvillaisaza** [comment]
Fixed after af5b7c3 and version 0.2.0.0.

@bergmark Should we do anything else about https://github.com/fpco/stackage/commit/02906f9f7cc105344aae50f279001c5f9c00e854? Thank you.
